pyrasterframes/src/main/python/pyrasterframes/__init__.py

```diff
@@ -249,6 +252,24 @@ def temp_name():
         .format("raster") \
         .load(path, **options)
 
+def _stac_api_reader(
+        df_reader: DataFrameReader,
+        uri: str,
+        filters: dict = None,
+        search_limit: Optional[int] = None) -> DataFrame:
+    """
+    uri - STAC API uri
+    filters - a STAC API Search filters dict (bbox, datetime, intersects, collections, items, limit, query, next)
+    search_limit - search results convenient limit method
```

Comment:
This might be better as named parameters, or as `**kwargs`.

Reply:
Discoverability when working in a notebook or something would benefit from these parameters being fully spelled out with defaults.